tests/src/common/Wsk.scala

```diff
@@ -767,7 +768,7 @@ sealed trait RunWskCmd {
         val args = baseCommand
         if (verbose) args += "--verbose"
         if (showCmd) println(params.mkString(" "))
-        val rr = TestUtils.runCmd(DONTCARE_EXIT, workingDir, TestUtils.logger, env, args ++ params: _*)
+        val rr = TestUtils.runCmd(DONTCARE_EXIT, workingDir, TestUtils.logger, sys.env ++ env, args ++ params: _*)
```

Comment:
system.basic.CLIJavaTests > Java Actions should invoke a java action STANDARD_OUT
    /home/vagrant/openwhisk/ansible/../bin/go-cli/wsk --apihost 172.17.0.1 --apiversion v1 action delete -i --auth 23bc46b1-71f6-4ed5-8c54-816aa4f8c502:123zO3xZCLrMN6v2BKK1dXYFpXlPkccOFqm12CdAsMgRU4VrNZ9lyGVCGuMDGIwP /_/helloJava
    /home/vagrant/openwhisk/ansible/../bin/go-cli/wsk --apihost 172.17.0.1 --apiversion v1 action create -i --auth 23bc46b1-71f6-4ed5-8c54-816aa4f8c502:123zO3xZCLrMN6v2BKK1dXYFpXlPkccOFqm12CdAsMgRU4VrNZ9lyGVCGuMDGIwP /_/helloJava /home/vagrant/openwhisk/ansible/../catalog/samples/helloJava/build/libs/helloJava.jar
    expected exit code = 0
    exit code = 1
    stdout: error: Unable to parse action command arguments: exec: "jar": executable file not found in $PATH
    Run 'wsk --help' for usage.